unlock-app/src/components/helpers/Balance.js

```diff
@@ -7,15 +7,19 @@ import Web3Utils from 'web3-utils'
  * @param {*} amount: the amount to convert to Eth
  * @param {string} unit: the unit of the amount to convert to Eth
  */
-export function Balance({ amount, unit = 'wei' }) {
-  const inWei = Web3Utils.toWei(amount || '0', unit)
-  const inEth = Web3Utils.fromWei(inWei, 'ether')
-  return (<span>Ξ {inEth}</span>)
+export function Balance({ amount, unit = 'wei', symbol=true }) {
```

Comment:
Do you mind adding a test for this? (I think the Balance component has tests?)

Reply:
👍